modules/cloudtrail/cmdb-lambda.tf

```diff
@@ -0,0 +1,8 @@
+module "cmdb" {
+  source = "./modules/cmdb-integration"
+  count  = var.cmdb_integration.enabled ? 1 : 0
+
+  name             = var.name
+  bucket_name      = local.s3_bucket_name
+  cmdb_integration = var.cmdb_integration
```

Comment:
here it seems better to simply have the following with together the module variable change:
```
  config = var.cmdb_integration.config